lib/settings-in-redis/settings.rb

```diff
@@ -61,7 +63,11 @@ def self.redis_key(var_name)
   # get or set a variable with the variable as the called method
   def self.method_missing(method, *args)
     if self.respond_to?(method)
+      # :nocov: I can't figure out how this is reachable - if an object responds to a method, how
+      # would .method_missing be called? - so I can't cover it in testing.  I'm afraid to rip it
+      # out, though.
```

Comment:
:yikes: yeah, that makes no sense.  Also, what's `super` in this case, the base `Object`?  Given that, I'd agree on not ripping it out, who knows what kinda magic that's performing.

Reply:
Yes, I think it is `Object`.

![yikes](https://media.giphy.com/media/l2JdUolkmRLmZM9Fu/giphy.gif)